talkitover_app.py

```diff
@@ -425,6 +447,7 @@ def choose_bot_wordy_response(message, clientId):
     feelingDepressedArray = ["i feel depressed", "i just feel depressed", "i feeel depressed", "i just feeel depressed", "im feeling depressed", "i'm feeling depressed", "im just feeling depressed", "i'm just feeling depressed", "i feel very depressed", "i feel so depressed", "i am feeling depressed", "i'm feeling really depressed", "i'm feeling so depressed", "im feeling really depressed", "im feeling so depressed",\
                             "i'm depressed", "im depressed", "i am depressed", "i'm so depressed", "im so depressed", "i am so depressed", "i'm really depressed", "im really depressed", "i am really depressed",\
                             "im more depressed than ever", "i'm more depressed than ever" ]
+    iHaveDepressionArray = ["I have depression"]
```

Comment:
This does not seem to be producing the required response when I run the code - it just elicits an 'encouraging noise response'. I think this is because there is a capital letter in the string.